pinot-tools/src/main/java/org/apache/pinot/tools/perf/QueryRunner.java

```diff
@@ -944,6 +948,16 @@ public double getAvgClientTime() {
       return _avgClientTime;
     }
 
+    public double getPercentile(double p) {
```

Comment:
maybe we don't need this method, as we have this method below, and we can get any percentile we'd need from the list of stats.
```
public List<Statistics> getStatisticsList() {
      return _statisticsList;
    }
```

Reply:
The Statistics class is a private one. 
Having the public getPercentile() can allow external usage of the percentile stats.